pydfs_lineup_optimizer/lineup_exporter.py

```diff
@@ -36,3 +37,37 @@ def export(self, filename, render_func=None):
                 row.append(str(lineup.salary_costs))
                 row.append(str(lineup.fantasy_points_projection))
                 lineup_writer.writerow(row)
+
+
+class JSONLineupExporter(LineupExporter):
+    def export(self):
+        totalLineups = []
+
+        for lineup in enumerate(self.lineups):
+            lineupList = []
+
+            # Generate players JSON object
+            for player in lineup.lineup:
```

Comment:
You can iterate over lineup without lineup property, like `for player in lineup:`